docs/source/release_notes/release-notes.rst

```diff
@@ -42,8 +42,11 @@ develop
     *    - Type
          - Change
 
-    *    -
-         -
+    *    - |improved| |devbreak|
+         - Improved performance by preventing excessive reads from the _namespace table when initializing SweepStrategyManager.
```

Comment:
@gbonik Thanks a lot for clarifying. This makes sense, merging for next release.